packages/web-api-scan-request-sender/src/sender/on-demand-scan-request-sender.ts

```diff
@@ -26,12 +33,16 @@ export class OnDemandScanRequestSender {
                 }
 
                 if (isEnqueueSuccessful === true) {
```

Comment:
move this inside the previous if block. & make the isEnqueueSuccessful a local variable to the previous if block.